wiki/Submission/en.md

```diff
@@ -34,91 +39,88 @@ You can also select the following via checkboxes under the message input box:
 -   **Receive email notification on reply:** When this box is checked, once someone replies to your beatmap's thread, you will receive a notification e-mails in the e-mail address you provided when you registered in osu!. You will receive no further notifications for any more replies until you visit the thread.
 -   **Load in browser after submission:** If this box is checked, after the upload is complete and you click "Submit", your default browser will attempt to display the page of your beatmap's forum thread.
 
-The Beatmap Modding Process
----------------------------
+### The Beatmap Modding Process
```

Comment:
imo, to prevent saying things multiple times, inconsistently, I'd merge the content of this section in with the Main article, then just say to go to that page ¯\_(ツ)_/¯